packages/discussions/src/utils/constants.ts

```diff
@@ -1,3 +1 @@
-export const CANNOT_DISCUSS = "cannotDiscuss";
```

Comment:
This would cause a breaking change for anything that currently imports `CANNOT_DISCUSS` from `@esri/hub-discussions`, which we should try to avoid. I'd instead import `CANNOT_DISCUSS` from `@esri/hub-common` here and simply re-export, preserving existing imports of this constant from `hub-discussions`, so we can publish these changes as a minor vs major.